horace_core/Tobyfit/tobyfit_DGfermi_resconv_init.m

```diff
@@ -24,29 +24,21 @@
 % ------
 %   win         Array of sqw objects, or cell array of scalar sqw objects
 %
-%  [Optional]
-%   indx        Pixel indices:
+% [Optional]
+%   ipix        Pixel indices for which the output is to be extracted from the
+%               sqw object(s)
 %
-%               Single sqw object:
-%               ------------------
-%                 - ipix            Array of pixels indices
-%            *OR* - {irun,idet,ien} Arrays of run, detector and energy bin index
-%                                   Dimension expansion is performed on scalar
-%                                  quantities i.e. each must be a scalar or array
-%                                  with arrays having the same length
+%               - Array of pixel indices. If there are multiple sqw objects,
+%                 it is then applied to every sqw object
 %
-%               Multiple sqw objects:
-%               ---------------------
-%                 - As above, assumed to apply to all sqw objects,
-%            *OR* - Cell array of the above, one cell array per sqw object
-%                  e.g. if two sqw objects:
-%                       {ipix1, ipix2}
-%                       {{irun1,idet1,ien1}, {irun2,idet2,ien2}}
+%               - Cell array of pixel indices arrays
+%                   - only one array: applied to every sqw object, or
+%                   - several pixel indices arrays: one per sqw object
 %
 %   opt         Option: 'tables' (default) or 'notables'
-%               If 'tables', then moderator and Fermi chopper lookup tables
-%              are added to the output argument lookup
-%
+%               If 'tables', then object_lookup tables for instrumnet
```

Comment:
```suggestion
%               If 'tables', then object_lookup tables for instrument
```

Reply:
Done (and also in tobyfit_DGdisk_resconv_init).